README.asciidoc

```diff
@@ -37,6 +37,18 @@ First of all, DON'T PANIC. It will take 5 minutes to get the gist of what Elasti
 * Run `bin/elasticsearch` on Linux or macOS. Run `bin\elasticsearch.bat` on Windows.
 * Run `curl -X GET http://localhost:9200/` to verify Elasticsearch is running.
 
+=== Other Run Possibilities (Linux)
+
+* Running Elasticsearch as a daemon and get its process ID in the file 'pidfile' : `bin/elasticsearch -d -p pidfile`
+
+* Running Elasticsearch with SysV init : `sudo -i service elasticsearch start`
+* Can be stopped using `sudo -i service elasticsearch stop`
+
+* Running Elasticsearch with systemd : `sudo systemctl start elasticsearch.service`
+* Can be stopped using `sudo systemctl stop elasticsearch.service`
+
+* You can configure SysV / systemd to run elasticsearch when your system boots up, see https://www.elastic.co/guide/en/elasticsearch/reference/current/starting-elasticsearch.html for more details.
```

Comment:
```suggestion
For more options, see
https://www.elastic.co/guide/en/elasticsearch/reference/current/starting-elasticsearch.html[Starting
Elasticsearch].
```

Reply:
Thanks for your answer ! Your suggestion is indeed more relevant to keep the quick start as clear as possible. Then, I can only approve that !